grommet/hpe-design-system
# Create reusable component to demo how to apply Tip to truncated Text in DataTable

#### What does this PR do?
Creates `TruncatedText` reusable component that will apply a tooltip to text when it has been truncated. Applies to appropriate DataTable examples.

#### Where should the reviewer start?
aries-site/src/examples/components/table/TruncatedText.js

#### What testing has been done on this PR?
Tested local in DS site.

#### How should this be manually tested?
In deploy, hover over a cell with "...", should see tooltip. For cell that isn't truncated, no tooltip should appear.

#### Any background context you want to provide?

#### What are the relevant issues?
Closes #1503 

#### Screenshots (if appropriate)

#### Should this PR be mentioned in Design System updates?
No.

#### Is this change backwards compatible or is it a breaking change?
Backwards compatible.
